darwin/foundation/nsdate.nim

```diff
@@ -0,0 +1,22 @@
+import times
+import ../objc/runtime
+
+
+type NSTimeInterval* = ptr float64
+type NSDate* = ptr object of NSObject
+
+
+proc objcClass*(t: typedesc[NSTimeInterval]): auto {.inline.} = objcClass("NSTimeInterval")
+proc objcClass*(t: typedesc[NSDate]): auto {.inline.} = objcClass("NSDate")
+
+
+proc newDate*(t: typedesc[NSDate]): NSDate {.objc: "date".}
```

Comment:
I think you don't have to define this. Just use `NSDate.new()`. Note that it will return an owned (to be released) object though.